core/channel.go

```diff
@@ -24,6 +27,10 @@ func CreateChannel(src, dst *ProvableChain, ordered bool, to time.Duration) erro
 	for ; true; <-ticker.C {
 		chanSteps, err := createChannelStep(src, dst, order)
 		if err != nil {
+			logger.Error(fmt.Sprintf("failed to create channel step: [%s]chan{%s}port{%s} -> [%s]chan{%s}port{%s}",
+				src.ChainID(), src.Path().ChannelID, src.Path().PortID,
+				dst.ChainID(), dst.Path().ChannelID, dst.Path().PortID),
+				zap.Error(err))
```

Comment:
You can use `Errorf(...)` instead of `Error(fmt.Sprintf(...))`, but anyway, neither of them are structured logging.
You need use `Errorw` here, like the following example, right?

```go
logger.Errorw("failed to create channel step"
    "source chain id", src.ChainID(),
    "source channel id", src.Path().ChannelID,
    ...,
    zap.Error(err))
```

Reply:
Use `Errorw` and Refactoring zap logger